src/System.Management.Automation/engine/CommandCompletion/PseudoParameterBinder.cs

```diff
@@ -357,7 +357,7 @@ public static StaticBindingResult BindCommand(CommandAst commandAst, bool resolv
         }
 
         [ThreadStatic]
-        static Runspace s_bindCommandRunspace = null;
+        private static Runspace s_bindCommandRunspace = null;
```

Comment:
Not for the PR - this should be `t_bindCommandRunspace`.

Reply:
see new PR: #12826